test/suites/ACID.m

```diff
@@ -2742,3 +2743,21 @@
   xlim([-1, 2]);
 end
 % =========================================================================
+function [stat] = colorbars()
```

Comment:
I know it is a pita but given that we allow different positions of the color bar i think we should test for all 4 cases.